manager/assets/modext/widgets/fc/modx.panel.fcset.js

```diff
@@ -49,7 +45,7 @@ MODx.panel.FCSet = function(config) {
                     ,value: config.record.action
                     ,listeners: {
                         'select': {scope:this,fn:function(f,e) {
-                            Ext.getCmp('modx-fcs-header').getEl().update(_('set')+': '+f.getRawValue());
+                            Ext.getCmp('modx-fcs-header').getEl().update(Ext.util.Format.htmlEncode(f.getRawValue()));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(f.getRawValue()));`